data/devShop.json

```
@@ -0,0 +1,86 @@
+        "text": "The Immersion Program is designed to mirror the real-world environment of a modern software company. In the first six weeks, students tackle intensive weekly Sprints, each focused on mastering key technologies and core engineering concepts — just like a professional dev team in action. This hands-on approach ensures that our graduates are not only proficient in the latest tools and technologies, but also understand how to apply them effectively in a team setting. By the end of the program, students have a solid foundation in software engineering principles and are ready to hit the ground running in their new careers."
+      },
+      {
+        "header": "Working with the latest technologies",
+        "text": "Sprints provide a deep working knowledge of the technology being introduced.",
+        "grid": [
+          "Data Structures",
+          "Classic Algorithms",
+          "Time Complexity",
+          "Relational Databases",
+          "Document Databases",
+          "RESTful API",
+          "Web servers (Node.js/express.js",
+          "Authentication + Authorization",
+          "Security",
+          "Micro Services",
+          "Cloud Deployment",
+          "Test Driven Development (TDD)",
+          "Continuous Integration (CI)",
+          "Continuous Deployment (CD)",
+          "HTML",
+          "CSS",
+          "UI/UX Design Patterns",
+          "JavaScript",
+          "Angular.JS",
+          "Angular / TypeScript",
+          "React",
+          "React Native",
+          "Ionic / Cordova",
+          "D3",
+          "Material UI",
+          "Bootstrap",
+          "Mobile Development"
+        ]
+      }
+    ]
+  },
+  "contractToHire": {
+    "title": "How Can Operation Spark Help My Company",
+    "subtitle": "Contract to Hire or Our Dev Shop",
+    "sections": [
+      {
+        "header": "Contract to Hire",
+        "text": "Need to quickly scale up your software team with proven talent? With Operation Spark's Contract-to-Hire Program, we provide and manage skilled developers who are ready to contribute from day one. After three months, if you're confident your project is moving forward, you have the option to bring our contractors on board as full-time employees. It's a “try-before-you-buy” model — flexible, low-risk, and effective. Let's talk about your resource needs and our competitive hourly rates."
+      },
+      {
+        "header": "Our Dev Shop - Hire Us",
+        "text": "Operation Spark develops custom software of all kinds and we're ready to build, maintain, and scale your next project. Best of all, we deliver high-quality results at a fraction of standard market rates, without compromising on performance or professionalism. Our client portfolio spans a wide range of projects, from simple maintenance to complex, large-scale solutions.",
+        "grid": [
+          "Two-year project for the USDA",
+          "Logistics applications for the shipping industry",
```

I don't know what this is referring to.
```suggestion
          "Logistics applications for the shipping industry",
```
```suggestion
```

---

removed + added stem + include links, if applicable https://github.com/OperationSpark/operationspark-org-website-2022/pull/206/commits/1fc467a6823abe873190fbfe9d1993b7c6e00145

---

This copy was for a different context, I think. It doesn't really make sense to me in the context of selling software development to companies